### PR TITLE
Release bootstrap image on quay.io

### PR DESCRIPTION
**Pull request by rmohr**
Work around strange push issues with docker.io.

Signed-off-by: Roman Mohr <rmohr@redhat.com>

### CODE REVIEW

File: github/ci/prow/files/jobs/kubevirt/project-infra/project-infra-postsubmits.yaml [hunk 1]
```diff
@@ -81,7 +81,10 @@ postsubmits:
               - "/usr/local/bin/runner.sh"
               - "/bin/bash"
               - "-c"
-              - "cat $DOCKER_PASSWORD | docker login --username $(<$DOCKER_USER) --password-stdin && cd images && ./publish_image.sh bootstrap docker.io kubevirtci"
+              - |
+                cat "$QUAY_PASSWORD" | docker login --username $(cat "$QUAY_USER") --password-stdin=true quay.io
+                cd images
+                ./publish_image.sh bootstrap quay.io kubevirtci"
```
Comment (fgimenez) [inline github/ci/prow/files/jobs/kubevirt/project-infra/project-infra-postsubmits.yaml:87]:
```suggestion
                ./publish_image.sh bootstrap quay.io kubevirtci
```

**Review by fgimenez — COMMENTED** [review]
Looks good, only a `"` remaining from the previous string

File: github/ci/prow/files/jobs/kubevirt/project-infra/project-infra-postsubmits.yaml (hunk 1 above)
Comment (rmohr) [reply github/ci/prow/files/jobs/kubevirt/project-infra/project-infra-postsubmits.yaml]: Done. :+1:

**Review by fgimenez — APPROVED** [review]
/lgtm
/approve

**Pull request merged**